pix-editor/app/components/pop-in/confirm-log.js

```diff
@@ -0,0 +1,16 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+
+export default class PopInConfirmLog extends Component {
+  @tracked displayTextarea = false;
+  @tracked defaultValue = this.args.defaultValue;
+  inputId = this.args.inputId;
```

Comment:
Est-ce utile de rendre cette Id dynamique sachant qu'on n’aura jamais deux pop-in ouverte en même temps ?

Reply:
Bien vu ::)